CMakeLists.txt

```diff
@@ -704,6 +709,25 @@ if(NOT HELICS_DISABLE_BOOST)
     )
 endif()
 
+# -----------------------------------------------------------------------------
+# create networking target
+# -----------------------------------------------------------------------------
+
+set(GMLC_NETWORKING_INSTALL OFF CACHE INTERNAL "")
+set(GMLC_NETWORKING_ASIO_INCLUDE ${PROJECT_SOURCE_DIR}/ThirdParty/asio/asio/include)
+set(GMLC_NETWORKING_CONCURRENCY_INCLUDE ${PROJECT_SOURCE_DIR}/ThirdParty/concurrency)
+add_subdirectory(ThirdParty/networking)
+
+hide_variable(GMLC_NETWORKING_GENERATE_DOXYGEN_DOC)
+hide_variable(GMLC_NETWORKING_INCLUDE_BOOST)
+hide_variable(GMLC_NETWORKING_WITH_CMAKE_PACKAGE)
+hide_variable(GMLC_NETWORKING_OBJECT_LIB)
+hide_variable(GMLC_NETWORKING_STATIC_LIB)
+hide_variable(GMLC_NETWORKING_CLANG_TIDY_OPTIONS)
+
+add_library(HELICS::networking ALIAS gmlc_networking)
```

Comment:
This sort of seems like gmlc_networking should export an alias target called GMLC::networking

Reply:
We might want to consider a refresh on all the submodule libraries in GMLC-TDC that would standardize some of this.  But this is probably a good idea, I want to make sure it is consistent with all of them though.